### PR TITLE
editorial

### PR DESCRIPTION
**Pull request by kaduk**
Well, hopefully, at least.
Trying to keep the easy stuff out of my ballot.

### CODE REVIEW

File: draft-ietf-suit-architecture.md [hunk 1]
```diff
@@ -300,7 +300,7 @@ The following stakeholders are used in this document:
 
 * Device Operator: The device operator is responsible for the day-to-day operation
   of a fleet of IoT devices. Customers of IoT devices, as the owners of
-  IoT devices - such as enterprise customers or end users, interact
+  IoT devices - such as enterprise customers or end users - interact
```
Comment (kaduk) [inline draft-ietf-suit-architecture.md:303]: This one I'm not sure I had the proper fix for -- there seem to be a lot of parenthetical phrases in here and I'm not sure how they're intended to fit together.

File: draft-ietf-suit-architecture.md [hunk 2]
```diff
@@ -671,15 +674,15 @@ implement the following functionality:
...
    firmware (such as to the status tracker).  This allows
-   to share information about the current firmware version, and the
+   sharing information about the current firmware version, and the
    status of the firmware update process and whether errors have occurred.
 
 -  Produce boot measurements as part of an attestation solution. See
    {{I-D.ietf-rats-architecture}} for more information. (optional)
 
--  Ability to decrypt firmware images, in case confidentiality protection
+-  The bootloader must be able to decrypt firmware images, in case confidentiality protection
```
Comment (kaduk) [inline draft-ietf-suit-architecture.md:685]: These were an attempt to keep the list structure paralle, but I'm not sure I quite got it right for all the list items.

File: draft-ietf-suit-architecture.md (hunk 1 above)
Comment (russhousley) [reply draft-ietf-suit-architecture.md:303]: This looks right to me.

File: draft-ietf-suit-architecture.md (hunk 2 above)
Comment (russhousley) [reply draft-ietf-suit-architecture.md:685]: I think this works.

**Pull request merged**